eggplant_project/settings/base.py

```diff
@@ -15,6 +15,10 @@
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.messages import constants as messages_constants
 
+COOP_NAME = 'KBHFF'
```

Comment:
Yes it's a design pattern. You put settings in the application in a single module, and then through well-known property names of django.conf.settings, a project can overwrite them.

Settings have to be centralized so they're easy to grasp, document, manipulate for testing etc.

I've tried this so many times, having settings randomly ad-hoc invented will degenerate the code.. that's why I'm saying that it's much better to create eggplant.settings or eggplant.membership.settings and have the design pattern in place from the beginning.